sway/input/cursor.c

```diff
@@ -835,7 +835,7 @@ static void handle_pointer_constraint_set_region(struct wl_listener *listener,
 		wl_container_of(listener, sway_constraint, set_region);
 	struct wlr_pointer_constraint_v1 *constraint = data;
```

Comment:
Let's remove these two declarations.

Reply:
Ah yes, those were there to test the wlroots patch version... removed.